CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
+### Added
+- [Hold Sideband PHY RESET low on older Cynthions](https://github.com/greatscottgadgets/apollo/pull/66)
```

Comment:
I think this should be listed as either `Fixed` or `Changed`, but definitely not `Added`.

Reply:
Works for me, pushed the change!